stock_scanner_web/views/web_templates.xml

```diff
@@ -0,0 +1,247 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    Copyright 2016 Angel Moya <http://angelmoya.es>
+    Copyright 2016 Eficent Business and IT Consulting Services, S.L.
+    <http://www.eficent.com>
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+
+<openerp>
+    <data>
+
+        <template id="stock_scanner_web.assets_frontend" name="Website assets">
+            <meta name="viewport"
+                  content="width=device-width, initial-scale=1"/>
+            <!-- Include the jQuery library -->
+            <script type="text/javascript"
+                    src="https://code.jquery.com/jquery-1.11.3.min.js"/>
```

Comment:
These externals libraries should be embedded into the module since in some cases the servers do not have access to internet